<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsMenu</class>
 <widget class="QDialog" name="SettingsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>141</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Card Back Texture</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Card Front Texture</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Background Texture</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Music</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>141</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="card_back_combo">
      <item>
       <property name="text">
        <string>card_back_1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>card_back_2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>card_back_3</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="card_front_combo">
      <item>
       <property name="text">
        <string>card_front_1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>card_front_2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>card_front_3</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="bg_combo">
      <item>
       <property name="text">
        <string>bg_1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bg_2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bg_3</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>107</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="music_on">
      <property name="text">
       <string>On</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="music_off">
      <property name="text">
       <string>Off</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>175</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="save_button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>82</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="back_preview_button">
      <property name="text">
       <string>Preview</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="front_preview_button">
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bg_preview_button">
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
